### ISSUE TITLE
legislators endpoint test is brittle

### ISSUE THREAD

**Issue by mchandleraz** [bstil22/govtracker#2]
I think it's because the test is looking for what appears to be a profile id in the response. if a delegate loses their seat, the response will have a new profile id (bioguide_id).

Maybe there's another way to assert that everything's good, without using an id?

**Comment by bstil22** [comment]
yeah could use regex matching, I think these ID's have a consistent pattern
